clevertap-core/src/test/java/com/clevertap/android/sdk/CTXtensionsTest.kt

```diff
@@ -1047,6 +1050,16 @@ class CTXtensionsTest : BaseTestCase() {
         assertFalse(result)
     }
 
+    @Test
+    fun `string format1`(){
```

Comment:
Those two tests seem redundant

Reply:
It's meant for testing `String.format()` usage in our SDK. `String.format()` has some breaking changes in Android 15.

Reply:
It seems a bit strange to me to test the SDK methods. Maybe we should test this with the methods where it is used in our code and verify that they still work correctly or wrap the String.format in our own function and test that. If this is the only way to test this, maybe at least compare the result of .format() with assertEquals to the expected behavior.

Reply:
fixed in https://github.com/CleverTap/clevertap-android-sdk/commit/5b7c660be6474a5204cf93ba2e6f97a86869ebf4